backend/functions/lib/use-cases/case-docket/case-docket.ts

```diff
@@ -0,0 +1,18 @@
+import { CaseDocketGateway } from '../../adapters/gateways/gateways.types';
+import { ApplicationContext } from '../../adapters/types/basic';
+import { CaseDocket } from './case-docket.model';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const MODULE_NAME = 'CASE-DOCKET-USE-CASE';
```

Comment:
Feels like we should keep this around for consistency, but it ISN'T used. Delete?